app/controllers/api/services_controller.rb

```diff
@@ -34,20 +34,17 @@ def settings
   end
 
   def create
-    @service, success = create_or_build_service
+    @service = collection.new # this is done in 2 steps so that the account_id is in place as preffix_key relies on it
+    @service.attributes = params[:service]
```

Comment:
`params.require(:service)` and so on?

Reply:
That's a rollback. Not really touching the old code here.

Reply:
Ah wait did you reverted back to before the service discovery?

Reply:
👍

Reply:
👍 It would have been good to mention it in the commit messages then :)

Reply:
Done